app/src/componentLibrary/fieldTextFlex/README.md

```diff
@@ -0,0 +1,24 @@
+## **FieldTextFlex**
+
+Height has 72px minimum. Max height flexible by content.
+Width 100% from parent.
+
+### Props:
+
+- **value**: _string_, optional, default = ""
+- **placeholder**: _string_, optional, default = ""
+- **label**: _string_, optional, default = ""
+- **helperText**: _string_, optional, default = ""
```

Comment:
As by [comment](https://github.com/reportportal/service-ui/pull/3105#discussion_r879693150) let's rename it to `helpText` here and in the component to be more consistency across the components